.circleci/config.yml

```diff
@@ -181,7 +181,7 @@ jobs:
             sapi/cli/php run-tests.php \
               -d zend_extension=opcache.so \
               -d opcache.enable_cli=1 \
-              -d opcache.jit_buffer_size=16M \
+              -d opcache.jit_buffer_size=64M \
```

Comment:
Do we actually run into this limit? 16M is quite a lot for a test.

Reply:
As per the other config, I'd like to use the same parameters everywhere, to avoid hitting any limit and not compiling any trace, in case the unit tests were to include some bigger integration tests in the future.